<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomWindow</class>
 <widget class="QMainWindow" name="RoomWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoomWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
border-image: url(:/my_image/timyg.jfif);
}</string>
      </property>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>120</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/openled_ico.png);
border-image: url(:/image/room/closeled_ico.png);</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>130</y>
         <width>91</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/openled_ico.png);
border-image: url(:/image/room/closeled_ico.png);</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/ON.png);
border-image: url(:/image/room/OFF.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>320</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/image/room/closefan_ico.png);
border-image: url(:/image/room/openfan_ico.png);</string>
       </property>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>310</y>
         <width>91</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/my_image/Door.png);
border-image: url(:/image/room/opendoor_ico.png);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>返回主菜单</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>220</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/ON.png);
border-image: url(:/image/room/OFF.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>430</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/OFF.png);
border-image: url(:/image/room/ON.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>420</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/room/OFF.png);
border-image: url(:/image/room/ON.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
